### ISSUE TITLE
Read and Write at the same time

### ISSUE THREAD

**Issue by paulerolland** [xxxajk/spi4teensy3#6]
Hi Andrew,

First of all, thank you so much for this library that works great!
I am having an issue because I would like to read and write the SPI bus on the same clock. I tried to add a function to your code but it seems that I don't quite understand it as much as I need to.
Do you know how I could implement it?

Thanks for your help.

Paul

**Comment by eirikwor** [comment]
@paulerolland I would be very interested in that modification, if you still have it.

**Comment by xxxajk** [comment]
I've not seen any activity in a couple years, no idea what his problem was in getting that to go, but it should be very simple to tweak.

**Comment by paulerolland** [comment]
@eirikwor I will try to push my modification today. I thought I did it back then, it maybe did not work.

**Comment by eirikwor** [comment]
Thanks!

**Comment by cinderblock** [comment]
@paulerolland Did you end up getting this merged into this repo? I see the branch you pushed to your fork but I don't see that commit in this repository.

https://github.com/paulerolland/spi4teensy3/commit/4c1000c4f5cad593d04eb5aff7efe11ef2d948ad

**Comment by xxxajk** [comment]
Exactly, and I won't do any kind of update from another fork unless it is submitted.
